### PR TITLE
created internal user mock

### PR DESCRIPTION
**Pull request by dcdarrell9**
**What is the context of this PR?**

[card](https://trello.com/c/UPRGrQhE)

Added an internal user service mock and toggle

**How to review**

Ensure service runs and there are no errors

### CODE REVIEW

File: secure_message/api_mocks/internal_user_service_mock.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    @staticmethod
+    def get_user_details(uuid):
+        """gets the user details from the internal user service"""
+        logger.debug("getting mock user details for uaa")
+        if uuid:
+            internal_user_dict = {"id": uuid,
+                                  "firstName": "fred",
+                                  "lastName": "flinstone",
+                                  "emailAddress": "mock@email.com"}
+            return internal_user_dict, 200
```
Comment (necrophonic) [inline secure_message/api_mocks/internal_user_service_mock.py]: Do you need to assign `internal_user_dict` before returning it or could you return it directly without the assignment?

File: tests/behavioural/features/steps/common.py [hunk 2]
```diff
@@ -44,6 +46,12 @@ def step_impl_use_mock_case_service(context):
     case_service.use_mock_service()
 
 
+@given("using mock internal user service")
+def step_impl_use_mock_internal_user_service(context):
+    """ use the default party service tests"""
```
Comment (necrophonic) [inline tests/behavioural/features/steps/common.py]: 🎨 Minor cosmetic - consistency - should doc strings start with upper or lower case chars? Seems to have a mix here.

Comment (dcdarrell9) [reply tests/behavioural/features/steps/common.py]: Should be done right 👍

File: secure_message/api_mocks/internal_user_service_mock.py (hunk 1 above)
Comment (AndrewTorrance) [reply secure_message/api_mocks/internal_user_service_mock.py]: This needs to accept a uuid and select the relevant dictionary entry . I think those tests we discussed yesterday will help ?

**Pull request merged**